src/SwedbankPay/Core/Library/Methods/Checkout.php

```diff
@@ -10,6 +10,48 @@
 
 trait Checkout
 {
+	/**
+	 * Check Checkout API Credentials.
+	 *
+	 * @return bool Returns true if ok
+	 * @throws Exception
+	 */
+	public function checkCheckoutApiCredentials()
+	{
+		/**
+		 * Perform a POST request to create a payment or paymentorder with invalid data that is known to fail with
```

Comment:
The comment here says `payment or paymentorder`, but the request only performs a request towards `paymentorder`.